doc/man1/openssl-s_server.pod.in

```diff
@@ -426,19 +438,55 @@ option is enabled the peer does not need to send the close_notify alert and a
 closed connection will be treated as if the close_notify alert was received.
 For more information on shutting down a connection, see L<SSL_shutdown(3)>.
 
+=item B<-servername>
+
+Servername for HostName TLS extension.
+
+=item B<-servername_fatal>
+
+On servername mismatch send fatal alert (default: warning alert).
+
 =item B<-id_prefix> I<val>
 
 Generate SSL/TLS session IDs prefixed by I<val>. This is mostly useful
 for testing any SSL/TLS code (e.g. proxies) that wish to deal with multiple
 servers, when each of which might be generating a unique range of session
 IDs (e.g. with a certain prefix).
 
+=item B<-keymatexport>
+
+Export keying material using label.
+
+=item B<-keymatexportlen>
+
+Export len bytes of keying material; default 20.
```

Comment:
`I<len>` I think, although the synopsis has `+int`, again update the synopsis.  Perhaps `length` might be better.

Reply:
Ah, I had mostly taken over the texts from the help strings of the apps,
and at a couple of points like this did not notice that some adaptations were due.
Changed to:
`Export the given number of bytes of keying material`